command.go

```diff
@@ -224,6 +236,18 @@ func (c *Command) runWithResult(cmd *exec.Cmd) (*Result, error) {
 // and returns the results (stdout/stderr, exit code, etc). args should be the
 // URLs that would normally be passed in to yt-dlp.
 func (c *Command) Run(ctx context.Context, args ...string) (*Result, error) {
+	if c.progressFunc != nil {
+		c.Progress()
+		c.ProgressTemplate("dl:{\"id\":\"%(info.id)s\",\"playlist_id\":\"%(info.playlist_id)s\",\"title\":\"%(info.title)s\",\"status\":\"%(progress.status)s\",\"total_bytes\":\"%(progress.total_bytes)s\",\"total_bytes_estimate\":\"%(progress.total_bytes_estimate)s\",\"downloaded_bytes\":\"%(progress.downloaded_bytes)s\",\"speed\":\"%(progress.speed)s\",\"percent_str\":\"%(progress._percent_str)s\",\"playlist_count\":\"%(info.playlist_count)s\",\"playlist_index\":\"%(info.playlist_index)s\"}\n")
```

Comment:
Can we be confident that some title might not have one of:

- newlines (how would this be handled?)
- quote symbols and/or something that JSON doesn't directly support?

I am not super familiar with yt-dlp and it's escaping rules for `%(field)s` type fields.

To further clarify on my previous comments, I don't necessarily think the format has to be JSON, just handled in the same way that timestamp-based logging happens. I'm not sure if JSON here is better or worse (assuming there is potential for weird characters to cause JSON to break).

Reply:
You are correct! Fortunately, yt-dlp supports printing fields with escaped double quotes using the %()j formatting option. I will update the template to utilize this option for proper double-quote handling.